src/main/java/tc/oc/pgm/blockdrops/BlockDropsMatchModule.java

```diff
@@ -52,9 +52,10 @@
   // die that do not fire an event e.g. the tick age limit, but this should be
   // rare and they will only leak until the end of the match.
   private final Set<FallingBlock> fallingBlocksThatWillNotLand = new HashSet<>();
+  private final Match match;
```

Comment:
This `match` field will now pop up in each `MatchModule` implementation. We should have `MatchModule.Impl` with this to keep it DRY.

Reply:
There are some `MatchModule`s that don't even need a `Match` reference. They just use event listeners. Not really nessecary.